### ISSUE TITLE
Insertion without a model

### ISSUE THREAD

**Issue by lexkrstn** [uptrace/bun#739]
How to use bun as a query builder to insert a row into the database?

When I try this:

```go
_, err = r.db.NewInsert().
	Table("custom_path").
	Value("port_a_id", portAId).
	Value("port_b_id", portBId).
	Value("checkpoints", strings.Join(checkpoints, ",")).
	Value("path", "st_geomfromgeojson(?)", pathJson).
	On("conflict (port_a_id, port_b_id, checkpoints) do update").
	Set("path = excluded.path").
	Exec(r.ctx)
```

I get an error "bun: Model(nil)".

**Comment by vmihailenco** [comment]
Looks like a bug.

**Comment by moutoum** [comment]
Hi, I'd be very interested by this feature aswell. Do you have a plan to implement this?
Thanks

**Comment by i-fix-retarded-bugs** [comment]
Still not fixed? Well

**Comment by your-diary** [comment]
Any updates?

**Comment by j2gg0s** [comment]
> _, err = r.db.NewInsert().
> 	Table("custom_path").
> 	Value("port_a_id", portAId).
> 	Value("port_b_id", portBId).
> 	Value("checkpoints", strings.Join(checkpoints, ",")).
> 	Value("path", "st_geomfromgeojson(?)", pathJson).
> 	On("conflict (port_a_id, port_b_id, checkpoints) do update").
> 	Set("path = excluded.path").
> 	Exec(r.ctx)

Why not use raw query?
```go
db.NewRaw("INSERT INTO custom_path(port_a_id, port_b_id) VALUES(?, ?)", portAId, portBId).Exec(ctx)
```

**Comment by your-diary** [comment]
@j2gg0s 

> Why not use raw query?

I agree the example in the OP is a bit too simplified, but in real-world situations, I believe ORM is often utilized to dynamically (conditionally) construct SQL queries:

```go
query := db.NewInsert().Table("some_table")
if someCondition {
    query = query.Value(...)
}
if anotherCondition {
    query = query.Value(...)
}
query.Exec(...)
```

So, while `NewRaw()` is useful, I think it does not solve the core point of this issue.

**Comment by j2gg0s** [comment]
From the example, it seems more like a SQL Builder rather than an ORM.

On the other hand, the core of Bun is the Model. There is no way to generate SQL without relying on the Model.
